### DISCUSSION TITLE
Reverse DNS not working with UDM Pro

### DISCUSSION THREAD

**Discussion by rcdailey** [AdguardTeam/AdGuardHome#5737] [Q&A]
I have client settings set up so that I can identify each of my subnets and apply rules to them individually (screenshots at the bottom).

Even though I have my private reverse DNS setting set to `192.168.1.1` (my UDM router), I do not see individual client names in the query logs. Instead, the name I see is the name assigned under "persistent clients" which I think is wrong. I want it to use the client names from rDNS.

How can I set this up properly?

![image](https://user-images.githubusercontent.com/1768054/232230371-ea41ded6-05b9-4d1b-ae00-6965982e3e4d.png)

![image](https://user-images.githubusercontent.com/1768054/232230443-0c433c06-90c0-47da-8c0e-68ef1a30f5f2.png)

**Comment by ainar-g** [comment]
This is working as currently intended.  Once a persistent client is detected, their information shadows the rest.  This may change in the future, but there will always be a disparity between how much is known about the client and how much can be shown in a small table like that.

**Reply by rcdailey** [reply]
Thanks for the clarification. While it may be working as intended, I view this as an area in need of improvement because a user must now sacrifice client PTR lookups when they want to apply client-specific settings, such as blocked services.

I think it would make more sense to use the client settings name as a fallback if the PTR queries are disabled or those queries otherwise do not return a usable result.

**Reply by rcdailey** [reply]
Issue created here: https://github.com/AdguardTeam/AdGuardHome/issues/5741
